core/src/main/java/org/arquillian/smart/testing/api/SmartTesting.java

```diff
@@ -0,0 +1,15 @@
+package org.arquillian.smart.testing.api;
```

Comment:
does it really matter? :-)
Actually, you don't build anything - you configure and then invoke...

Reply:
`api.configuration` then? I'm not sure if `api` should be one big bag for everything

Reply:
`api.configuration` then? I'm not sure if `api` should be one big bag for everything

Reply:
So, what would you expect from the content of `api` package if we moved all the interfaces to the `api.configuration`. Or do you want to move only some of them... I'm slightly missing what you want to change. 

> one big bag for everything

Com'on, there are four classes/interfaces and I don't expect that it will grow significantly. If there was 10 of 15 of them, then I would say any word, but four?

Reply:
It's not really about amount but boundaries, however if you disagree then we have 1:1 and I would say we leave it.